src/main/msp/msp.c

```diff
@@ -1809,16 +1809,12 @@ case MSP_NAME:
 
```

Comment:
Define new function that will return index of gyro to report. `gyroConfig()->gyro_to_use == GYRO_CONFIG_USE_GYRO_ALL ? 0 : gyroConfig()->gyro_to_use;` is repeating multiple times below (and logic can be improved later)